paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/DefaultEventReporter.kt

```diff
@@ -91,6 +93,7 @@ internal class DefaultEventReporter @Inject internal constructor(
         linkDisplay: PaymentSheet.LinkConfiguration.Display,
         currency: String?,
         initializationMode: PaymentElementLoader.InitializationMode,
+        financialConectionsAvailability: FinancialConnectionsAvailability?,
```

Comment:
```suggestion
        financialConnectionsAvailability: FinancialConnectionsAvailability
```

Reply:
fixed the typo! but it still needs to be nullable (`null` meaning no availability)